spec/checkPullRequestLabelsSpec.js

```diff
@@ -793,9 +793,12 @@ describe('Pull Request Label Check', () => {
           color: '00FF00',
         };
 
+        const assigned = [{ login: 'SAEb-ai' }];
```

Comment:
Could we use a more generic login here? So that it is not linked to your username?

Reply:
Done